### ISSUE TITLE
jexcel script loading only once

### ISSUE THREAD

**Issue by Suragala** [jspreadsheet/ce#477]
Hi Paul,

In my application, I am using jexcel.

The jexcel scripts are loaded fine for the first time the page loads.
When I load the page again by selecting the same in the navigation bar, the jexcel scripts are not loaded.
Please let me know how to resolve the same.

**Comment by pphod** [comment]
Your description gives no clue what happened in your environment or what you are trying to achieve. Can you share a jsfiddle?

**Comment by jjoker0110** [comment]
I'm having this issue as well, I shimmed it into an AMD framework (require.js) and it works correctly the first time it loads but then after leaving the page and coming back it doesn't load the second time. 

I opened the terminal and run the command and don't get any errors but still nothing happens the after the first time ran. 

Still trying to debug through it and see if I can find where it is failing.
